src/app.ts

```diff
@@ -930,25 +928,31 @@ export class App {
 
   private writeResourceTests(
     scope: TypescriptTextWriter,
-    api: gapi.client.discovery.RestDescription,
+    api: RestDescription,
     ancestors: string,
     resourceName: string,
     resource: gapi.client.discovery.RestResource
   ) {
     _.forEach(resource.methods, (method, methodName) => {
       scope.comment(method.description);
       scope.newLine(`await ${ancestors}.${resourceName}.${methodName}(`);
-      const params:
-        | Record<string, gapi.client.discovery.JsonSchema>
-        | undefined = resource.methods![methodName].parameters;
+
+      const params: Record<string, JsonSchema> | undefined = method.parameters;
+      const ref = method.request?.$ref;
+
       if (params) {
         scope.scope(() => {
           this.writeProperties(scope, api, params);
         });
       }
-      const ref = method.request?.$ref;
-      if (ref != null) {
+
+      if (ref) {
+        if (!params) {
+          scope.write(`{} `);
```

Comment:
Arguably not the best solution but ensures the first argument is always passed in, even if there are no method-level parameters.